components/track/TrackChangeLocale.vue

```diff
@@ -0,0 +1,123 @@
+<script setup lang="ts">
+import { LanguageEnum } from "@bcc-code/bmm-sdk-fetch";
+import { Menu, MenuButton, MenuItem, MenuItems } from "@headlessui/vue";
+
+const expanded = ref(false);
+const { currentTrack, replaceCurrent } = useNuxtApp().$mediaPlayer;
+const { t } = useI18n();
+
+const changeLanguage = async (lang: LanguageEnum) => {
+  expanded.value = false;
+  if (!currentTrack.value) return;
+
+  try {
+    const reloadedTrack = await useTrackIDWithLanguage(lang, {
+      id: currentTrack.value.id,
+    });
+    if (!reloadedTrack.data.value) return;
+    replaceCurrent(reloadedTrack.data.value);
+  } catch (error) {
+    console.error(error);
+  }
+};
+
+const trackLanguages = currentTrack?.value?.languages || [];
+
+const getUserLanguages = () => {
+  const contentLangs = contentLanguageStore()
+    .contentLanguages as LanguageEnum[];
```

Comment:
We should change the contentLaguages to LanguageEnum[]. I should also not break things as we defined string constants for it.

Reply:
That won't work, because we do not know what is saved there. This comes into play when we go into backwards compatibility. `LanguageEnum` might change, but that won't change the content of what is saved in the store. Having it as a permanent store, we must not rely on it being compliant to the definition of `LanguageEnum`.